source/code-snippets/typescript/dot-notation.ts

```diff
@@ -19,7 +19,7 @@ interface TestNumber {
   myNumber: number;
 }
 
-const database = client.db("<your db>");
-const collection = db.collection<TestNumber>("...");
+const database = client.db("<your database>");
+const collection = db.collection<TestNumber>("<your collection>");
 collection.find({ someRandomKey: "Accepts any type!" });
```

Comment:
Oops, accidentally removed a comment I made here a few moments ago.
The comment was about whether the query relates to anything in the example. Would a document that includes a "TestNumber" contain a field called "someRandomKey"? Or maybe the example could be more relatable with field names that you might expect to find together.

Reply:
Updated to have the example tie more to something in the world.